.travis.yml

```diff
@@ -30,7 +17,7 @@ install:
     - pip install coveralls
 
 script:
-    - tox
+    - tox -e $TOXENV
```

Comment:
Is there a particular reason to add this change?

Reply:
removed.